src/pe_reports/data/db_query.py

```diff
@@ -0,0 +1,193 @@
+"""Query the PE postgres database."""
+
+# Standard Python Libraries
+import logging
+import sys
+
+# Third-Party Libraries
+import pandas as pd
+import psycopg2
+from psycopg2 import OperationalError
+from psycopg2.extensions import AsIs
+
+from .config import config
+
+logging.basicConfig(format="%(asctime)-15s %(levelname)s %(message)s", level="INFO")
+
+CONN_PARAMS_DIC = config()
+
+
+def show_psycopg2_exception(err):
+    """Handle errors for postgres issues."""
+    err_type, traceback = sys.exc_info()
+    line_n = traceback.tb_lineno
+    logging.error(f"\npsycopg2 ERROR: {err} on line number: {line_n}")
+    logging.error(f"psycopg2 traceback: {traceback} -- type: {err_type}")
+    logging.error(f"\nextensions.Diagnostivcs: {err}")
```

Comment:
Is this function necessary?  Would the extant [`logging.exception`](https://docs.python.org/3/library/logging.html#logging.Logger.exception) work just as well?

If this function stays, I think there is a typo here that should be fixed:
```suggestion
    logging.error(f"\nextensions.Diagnostics: {err}")
```